tests/containers/integration-test-local

```diff
@@ -19,4 +21,13 @@ RUN dnf install --repo bluechi-rpms \
 
 RUN dnf -y clean all
 
+RUN if [ "$with_coverage" = "1" ]; then \
+        dnf install -y --repo bluechi-rpms \
+        --repofrompath bluechi-rpms,file:///tmp/bluechi-rpms/ \
+        --nogpgcheck \
+        --nodocs \
+        bluechi-coverage && \
+    dnf -y clean all; \
```

Comment:
Ah, sorry, I missed that before :-(
We are cleaning DNF cache on line 20, so it would be better to move the IF section before that line and remove clean from IF section, so we clean only once